backend/corpora/dataset_processing/requirements.txt

```diff
@@ -0,0 +1,12 @@
+numba==0.53.1
```

Comment:
Nit: can we please alphabetize `requirements.txt` docs? Makes it a lot easier to read.